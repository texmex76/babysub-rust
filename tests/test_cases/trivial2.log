c BabySub Subsumption Preprocessor
c reading from '/home/gstrein/Documents/repos/babysub-rust/tests/test_cases/trivial2.cnf'
c parsed 'p cnf 2 2' header
c checked:                      0    clauses 0.00 per subsumed
c subsumed:                     0    clauses 0%
c process-time:                 0.00 seconds
