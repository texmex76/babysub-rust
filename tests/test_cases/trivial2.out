c BabySub Subsumption Preprocessor
c reading from '/home/gstrein/Documents/repos/babysub-rust/tests/test_cases/trivial2.golden'
c parsed 'p cnf 2 1' header
c tmp: 3450056017
c tmp: 2221466251909025616
c hash-signature: 1653562859555728976
p cnf 2 1
1 -2 0
c checked:                      0    clauses 0.00 per subsumed
c subsumed:                     0    clauses 0%
c process-time:                 0.00 seconds
